src/Microsoft.ML.StandardLearners/Microsoft.ML.StandardLearners.csproj

```diff
@@ -11,6 +11,7 @@
     <ProjectReference Include="..\Microsoft.ML.CpuMath\Microsoft.ML.CpuMath.csproj" />
     <ProjectReference Include="..\Microsoft.ML.Data\Microsoft.ML.Data.csproj" />
     <ProjectReference Include="..\Microsoft.ML\Microsoft.ML.csproj" />
+    <PackageReference Include="MlNetMklDeps" Version="$(MlNetMklDepsPackageVersion)" />
```

Comment:
This isn't going to work because the `Microsoft.ML` package doesn't reference the `MlNetMklDeps` package (and in my opinion, it should never reference it).

We will need to split OLS (and any other functionaltiy that depends on MKL) out into a separate NuGet package.  

In general, if a component takes a dependency on "non-standard" or "large" external dependencies, we can't ship that component in our "Main" nuget package.